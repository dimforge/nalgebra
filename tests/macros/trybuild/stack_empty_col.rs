use nalgebra::{matrix, stack};

fn main() {
    let m = matrix![1, 2; 3, 4];
    stack![0, m];
}
