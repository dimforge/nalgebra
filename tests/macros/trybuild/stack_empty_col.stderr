error: Block column 0 cannot consist entirely of implicit zero blocks.
 --> tests/macros/trybuild/stack_empty_col.rs:5:5
  |
5 |     stack![0, m];
  |     ^^^^^^^^^^^^
  |
  = note: this error originates in the macro `stack` (in Nightly builds, run with -Z macro-backtrace for more info)
